lib/x11regressiontest.pm

```diff
@@ -545,6 +545,22 @@ sub firefox_check_popups {
     }
 }
 
+sub firefox_open_url {
+    my ($self, $url) = @_;
+    my $counter = 1;
+    while (1) {
+        send_key 'alt-d';
+        send_key 'delete';
+        last if check_screen('firefox-empty-bar', 3);
+        if ($counter++ > 5) {
+            assert_screen('firefox-empty-bar', 0);
+            last;    # in case it worked
+        }
```

Comment:
how about

```perl
do {
    assert_screen([qw(empty not-empty)]);
} until match_has_tag …;
```

Reply:
there are too many values for not-empty - just check https://openqa.opensuse.org/tests/559359#step/firefox/5 - firefox change rate is just aweful